project/QuentinDucoulombier/README.md

```diff
@@ -0,0 +1,109 @@
+# (Project Proposal) DeliveryOptimizer: A Simplified Home-to-Home Delivery Optimization Tool
+
+## Basic Information
+
+DeliveryOptimizer aims to provide a simplified yet efficient solution for
+optimizing home-to-home delivery routes. The codebase will be hosted on GitHub
+at: https://github.com/QuentinDucoulombier/DeliveryOptimizer
+
+The project will be developed using a combination of C++ for computational
+efficiency and Python for ease of use and data handling.
+
+## Problem to Solve
+
+The primary challenge addressed by DeliveryOptimizer is the optimization of
+delivery routes for a single delivery vehicle, minimizing the total distance
+traveled while ensuring each home is visited exactly once. Unlike traditional
+delivery optimization problems, this simplified version does not consider
+vehicle capacity constraints or specific delivery time windows.
+
+## Prospective Users
+
+The target users of DeliveryOptimizer include:
+
+- Small to medium-sized enterprises (SMEs) operating in the e-commerce and
+logistics sectors that require a straightforward tool for planning delivery
+routes.
+
+## System Architecture
+
+DeliveryOptimizer will consist of two main components:
+
+1. C++ Backend: This component will handle route calculations and optimizations.
```

Comment:
This rough description is ok, but it does not provide a lot of help to your implementation.  You can consider to add prototype classes even though they are very likely to be changed when writing the code.  They will provide you insights about what may be needed.